src/main/java/jp/co/yahoo/yosegi/binary/ColumnBinaryMakerConfig.java

```diff
@@ -94,6 +94,7 @@ public ColumnBinaryMakerConfig() throws IOException {
     bytesMakerClass = FindColumnBinaryMaker.get( DumpBytesColumnBinaryMaker.class.getName() );
 
     compressionPolicy = CompressionPolicy.DEFAULT;
+    allowedRatio = 1.15d;
```

Comment:
I think that it is better not to specify this value directly but to be able to set it by the external setting file.
However, if this parameter realizes good performance, it seems useful to set an initial setting.